es-ds-components/components/es-dropdown.vue

```diff
@@ -0,0 +1,188 @@
+
+const isSelected = (option: any) => option === props.modelValue;
+
+const id = useId();
+</script>
+
+<template>
+    <div
+        class="input-wrapper justify-content-end"
+        :class="$attrs.class">
+        <label
+            v-if="label !== ''"
+            :for="id">
+            {{ label }}
+        </label>
+        <label
+            v-else
+            :for="id"
+            class="sr-only">
+            Select an option
+        </label>
+        <dropdown
+            :input-id="id"
+            :class="[
+                'es-dropdown-input d-flex align-items-center bg-white rounded-xs justify-content-between p-100',
+                {
+                    disabled: disabled,
+                    'es-dropdown-open': isOpen,
+                    focused: isFocused && !isOpen,
+                },
+            ]"
+            :disabled="disabled"
+            :focus-on-hover="false"
+            :model-value="modelValue"
+            :option-label="options.length > 0 && typeof options[0] === 'object' ? 'label' : undefined"
+            :options="options"
+            :placeholder="placeholder"
+            :pt="{
+                panel: { class: 'es-dropdown-panel bg-white rounded-xs' },
+                wrapper: { class: 'es-dropdown-wrapper' },
+                list: { class: 'p-0 m-0 list-unstyled' },
+                item: {
+                    class: [
+                        'es-dropdown-item d-flex justify-content-between p-100 pl-200',
+                        {
+                            'input-focused': isFocused,
+                        },
+                    ],
+                },
+            }"
+            scroll-height="18rem"
+            @update:model-value="emit('update:modelValue', $event)"
+            @blur="isFocused = false"
+            @focus="isFocused = true"
+            @click="isFocused = false"
+            @hide="isOpen = false"
+            @show="isOpen = true">
+            <template #dropdownicon>
+                <icon-chevron-down
+                    aria-hidden="true"
+                    height="1.125rem"
+                    :class="`${disabled ? 'text-gray-500' : 'text-gray-900'}`" />
+            </template>
+            <template #option="slotProps">
+                <span>{{ slotProps.option.label ? slotProps.option.label : slotProps.option }}</span>
+                <icon-check
+                    v-if="isSelected(slotProps.option)"
+                    height="1.5rem"
+                    class="text-gray-700" />
+            </template>
+        </dropdown>
+    </div>
+</template>
+
+<style lang="scss">
+@use '@energysage/es-ds-styles/scss/variables' as variables;
+
+.es-dropdown-input {
+    background-clip: padding-box;
+    border: variables.$border-width solid variables.$gray-900;
+    height: variables.$input-btn-height;
+    transition: variables.$input-transition;
+    user-select: none;
+
+    &:hover,
+    &:active {
+        border-color: variables.$blue-300;
```

Comment:
The text input's border becomes blue when it receives focus and stays blue throughout your interaction with it, until you're done and move on to another input.

https://github.com/user-attachments/assets/0cd1e481-b629-4bac-a8f3-18a6ca3d4451

Whereas the dropdown border currently only becomes blue on hover and goes back to being gray as soon as you move your mouse elsewhere or to the menu. I'd expect it to match the behavior of input, which is that the border becomes blue and stays blue during the entire duration of the time the user is engaging with the dropdown, meaning focused or menu open.

https://github.com/user-attachments/assets/79704436-b658-46b6-9121-f24983839411